backend/internal/service/handler/users/follow.go

```diff
@@ -0,0 +1,72 @@
+package users
+
+import (
+	"github.com/gofiber/fiber/v2"
+	"github.com/google/uuid"
+)
+
+type FollowRequest struct {
+	FollowerId  uuid.UUID `json:"followerId"`
```

Comment:
```suggestion
	FollowerId  uuid.UUID `json:"follower_id"`
```
super nit but camel case is our standard for json fields